content/docs/releases/enterprise/install/quickstart.mdx

```diff
@@ -1,232 +1,166 @@
-   audience: console.localhost.pomerium.com
-   ```
-
-   This sets the expected "audience" key in the [JWT header](/docs/reference/jwt-claim-headers) to match what's provided by open-source Pomerium as it proxies traffic to the Enterprise Console UI.
-
-### License Key
-
-Finally, add your license key to the configuration file:
-
-```yaml title="/etc/pomerium-console/config.yaml"
-license-key: YOUR_LICENSE_KEY
+```title="config-console.yaml"
+administrators: admin@example.com
+license_key: REPLACE_ME
 ```
 
----
-
-Once complete, your `/etc/pomerium-console/config.yaml` file should look something like this:
-
-```yaml title="/etc/pomerium-console/config.yaml"
-database_url: postgres://user:pass@dbhost.internal.mydomain.com/pomerium?sslmode=require
-databroker_service_url: https://pomerium-cache.internal.mydomain.com
-shared_secret: 'XXXXXXXXXXXXXXXXXXX'
-database_encryption_key: 'YYYYYYYYYYYYYYYYYYYYYY'
-
-# change / remove this after initial setup
-administrators: you@mydomain.com
-
-tls_ca_file: /etc/pomerium-console/ca.pem
-tls_cert_file: /etc/pomerium-console/cert.pem
-tls_key_file: /etc/pomerium-console/key.pem
-
-signing_key: 'ZZZZZZZZZZZZZZ'
-
-audience: console.localhost.pomerium.com
-
-license-key: YOUR_LICENSE_KEY
+You must configure at least one administrator for console access. This user (or users) can then configure additional administrators in the console UI. Multiple administrators are defined as a comma-separated string.
+
+## Configure Docker Compose services
+
+Create a file called `docker-compose.yaml` in your project's root folder and add the configuration below:
+
+```yaml title="docker-compose.yaml"
+version: '3'
+networks:
+  main: {}
+services:
+  pomerium:
+    image: pomerium/pomerium:v0.21.0
+    volumes:
+      - ./config.yaml:/pomerium/config.yaml:ro
+    ports:
+      - 443:443
+    networks:
+      main:
+        aliases:
+          - authenticate.localhost.pomerium.io
+    environment:
+      - AUTHENTICATE_SERVICE_URL=https://authenticate.localhost.pomerium.io
+      - COOKIE_SECRET=j9jZgysWVxCs3uqbmw9a2LxWwz1ZPLKQZ8v20eoDT8Y=
+      - SIGNING_KEY=LS0tLS1CRUdJTiBFQyBQUklWQVRFIEtFWS0tLS0tCk1IY0NBUUVFSURRemVZWDZyT2tuemFnTjRJVTYxaEtRc3pzY1EvRllmbzZPcXhWd2YvdGZvQW9HQ0NxR1NNNDkKQXdFSG9VUURRZ0FFc1V0V2psYXZ3eHprSU9DVUNDeFVnTDJza2NjL3QxSTFmQXlxUDgrMWw5YU1CWDlzdm1pYgpRajJxcWFUbUJZZWhuQzhmak5LODZmVXhpc3d1SXN5bnp3PT0KLS0tLS1FTkQgRUMgUFJJVkFURSBLRVktLS0tLQo=
+      - SHARED_SECRET=mxGl062SqkrbQKvqG9R2jqHqxq1Oi1BNj2AAeZHNq7c=
+      - DATABROKER_STORAGE_TYPE=postgres
+      - DATABROKER_STORAGE_CONNECTION_STRING=postgresql://postgres:postgres@database/postgres?sslmode=disable
+  pomerium_console:
+    networks:
+      main:
+    depends_on:
+      database:
+        condition: service_healthy
+      pomerium:
+        condition: service_started
+    image: docker.cloudsmith.io/pomerium/enterprise/pomerium-console:v0.21.0
+    args:
+      - 'serve'
+      - '--config'
+      - '/pomerium/console-config.yaml'
```

Comment:
When I try to do `docker compose up` I have an error `services.pomerium_console Additional property args is not allowed
`